03-rnaseq/differential-expression_rnaseq_01.Rmd

```diff
@@ -187,6 +187,14 @@ library(ggplot2)
 library(magrittr)
 ```
 
+The jitter plot we make later on involves some randomness. 
```

Comment:
Actually, it appears that we are not making a jitter plot later on in this notebook.

Reply:
We do, but it's made using `DESeq2::plotCounts()` function instead of `geom_jitter()` and it doesn't have an equivalent `height` argument (that I can find).

Reply:
https://github.com/AlexsLemonade/refinebio-examples/blob/28fb568c27ab531af6b77d11c83f42f8ed5e15bd/03-rnaseq/differential-expression_rnaseq_01.Rmd#L381

Reply:
But the output from this function, is still a jitter plot.

Reply:
Ahh of course, gotcha 👍

Reply:
It might be worth referring to the actual function that will generate the jitter plot to avoid confusion.